src/main/scala/de/zalando/beard/renderer/BeardTemplateRenderer.scala

```diff
@@ -12,12 +13,14 @@ class BeardTemplateRenderer(templateCompiler: TemplateCompiler) {
   def render[T](template: BeardTemplate,
                 result: RenderResult[T],
                 context: Map[String, Any] = Map.empty,
-                layout: Option[BeardTemplate] = None): T = {
+                layout: Option[BeardTemplate] = None,
+                escapeStrategy: EscapeStrategy = EscapeStrategy.html): T = {
```

Comment:
right not we should have the vanilla by default, as we shouldn't assume that we're rendering html by default